index.html

```diff
@@ -180,6 +185,22 @@
       </p>
 
       </section>
+
+      <section>
+      <h3>Processing incoming payment method data</h3>
+
+      <p>When the PaymentRequest API is to pass data to a payment app supporting basic-card as part
```

Comment:
Bit of an English problem here it looks like. I think it should be: "When the PaymentRequest API passes data to a payment app..."